src/librustc_save_analysis/span_utils.rs

```diff
@@ -409,7 +409,7 @@ impl<'a> SpanUtils<'a> {
 
         // Otherwise, a generated span is deemed invalid if it is not a sub-span of the root
         // callsite. This filters out macro internal variables and most malformed spans.
-        !parent.source_callsite().contains(parent)
+        !parent.source_callsite().contains(sub_span.unwrap())
```

Comment:
I'm weary of `unwrap()`. Even though it the code is valid now, somebody could come later and inadvertently remove the lines [401-403](https://github.com/rust-lang/rust/pull/43533/files#diff-8a330f51fa78f086da6ee70dc06a480dR401) without changing this. Could you change the above lines to `if let Some(sub_span)`, `else return true;`? I know it's a nitpick.